pallets/runtime/mainnet/src/runtime.rs

```diff
@@ -0,0 +1,378 @@
+    apis: RUNTIME_API_VERSIONS,
+    transaction_version: 7,
+};
+
+parameter_types! {
+    /// Assume 10% of weight for average on_initialize calls.
+    pub MaximumExtrinsicWeight: Weight = AvailableBlockRatio::get()
+        .saturating_sub(Perbill::from_percent(10)) * MaximumBlockWeight::get();
+    pub const Version: RuntimeVersion = VERSION;
+
+    // Frame:
+    pub const EpochDuration: u64 = EPOCH_DURATION_IN_BLOCKS as u64;
+    pub const ExpectedBlockTime: Moment = MILLISECS_PER_BLOCK;
+    pub const SS58Prefix: u8 = 12;
+
+    // Base:
+    pub const MaxLen: u32 = 2048;
+
+    // Indices:
+    pub const IndexDeposit: Balance = DOLLARS;
+
+    // Balances:
+    pub const ExistentialDeposit: Balance = 0u128;
+    pub const MaxLocks: u32 = 50;
+
+    // Timestamp:
+    pub const MinimumPeriod: Moment = SLOT_DURATION / 2;
+
+    // Authorship:
+    pub const UncleGenerations: BlockNumber = 0;
+
+    // Session:
+    // NOTE: `SessionHandler` and `SessionKeys` are co-dependent:
+    // One key will be used for each handler.
+    // The number and order of items in `SessionHandler` *MUST* be the same number
+    // and order of keys in `SessionKeys`.
+    // TODO: Introduce some structure to tie these together to make it a bit less of a footgun.
+    // This should be easy, since OneSessionHandler trait provides the `Key` as an associated type. #2858
+    pub const DisabledValidatorsThreshold: Perbill = Perbill::from_percent(17);
+
+    /*
+    // Contracts:
+    pub const NetworkShareInFee: Perbill = Perbill::from_percent(60);
+    pub const TombstoneDeposit: Balance = 0;
+    pub const RentByteFee: Balance = 0; // Assigning zero to switch off the rent logic in the contracts;
+    pub const RentDepositOffset: Balance = 300 * DOLLARS;
+    /// Reward that is received by the party whose touch has led
+    /// to removal of a contract.
+    pub const SurchargeReward: Balance = 150 * DOLLARS;
+    */
+
+    // Settlement:
+    pub const MaxLegsInInstruction: u32 = 10;
+
+    // Offences:
+    pub OffencesWeightSoftLimit: Weight = Perbill::from_percent(60) * MaximumBlockWeight::get();
+
+    // I'm online:
+    pub const SessionDuration: BlockNumber = EPOCH_DURATION_IN_SLOTS as _;
+    pub const ImOnlineUnsignedPriority: TransactionPriority = TransactionPriority::max_value();
+
+    // Assets:
+    pub const MaxNumberOfTMExtensionForAsset: u32 = 5;
+    pub const AssetNameMaxLength: u32 = 128;
+    pub const FundingRoundNameMaxLength: u32 = 128;
+
+    // Compliance manager:
+    pub const MaxConditionComplexity: u32 = 50;
+
+    // Corporate Actions:
+    pub const MaxTargetIds: u32 = 1000;
+    pub const MaxDidWhts: u32 = 1000;
+
+    // Statistics:
+    pub const MaxTransferManagersPerAsset: u32 = 3;
+
+    // Scheduler:
+    pub MaximumSchedulerWeight: Weight = Perbill::from_percent(80) * MaximumBlockWeight::get();
+    pub const MaxScheduledPerBlock: u32 = 50;
+
+    // Identity:
+    pub const InitialPOLYX: Balance = 100_000 * POLY;
```

Comment:
As nice as this would be I'm pretty sure this was an ITN specific config 😁
```suggestion
    pub const InitialPOLYX: Balance = 0 ;
```